.github/workflows/checks.yml

```diff
@@ -92,7 +94,10 @@ jobs:
       - uses: actions/checkout@v4
       - name: Set up environment
         uses: ./.github/actions/setup
-      - run: npm run coverage
+      - name: Compile contracts # TODO: Remove after migrating tests to ethers
+        run: npm run compile
```

Comment:
I don't think this is needed because the `solcover.js` config specifies that `compileCommand: 'npm run compile'`. Are you sure this actually did something?

Just curious, we can leave if it'll be removed at the end of the migration

Reply:
I'm not sure it actually does somthing. I just added that to all workflows

Lets see if we can remove them after the migration.